.rubocop.yml

```diff
@@ -104,6 +104,10 @@ Metrics/BlockLength:
 # Warns when the class is excessively long.
 Metrics/ClassLength:
   Max: 100
+  Exclude:
+    # Splitting the Generator classes would make matters more confusing
+    # than them being too big
+    - lib/generators/rspec/**/*
```

Comment:
Add the relevant magic comment to the generator that fails rather than doing this, or add the specific files, its possible to extract things without class splitting.